src/org/zaproxy/zap/extension/pscan/PassiveScanParam.java

```diff
@@ -51,6 +51,7 @@
 
     private static final String SCAN_ONLY_IN_SCOPE_KEY = PASSIVE_SCANS_BASE_KEY + ".scanOnlyInScope";
     private static final String SCAN_FUZZER_MESSAGES_KEY = PASSIVE_SCANS_BASE_KEY + ".scanFuzzerMessages";
+    private static final String MAX_ALERTS = PASSIVE_SCANS_BASE_KEY + ".maxAlerts";
```

Comment:
The key (and related variables/methods/) could more explicit, `maxAlertsPerRule`?

Reply:
Yeah, good suggestion.
Fixed